torch/utils/data/datapipes/datapipe.py

```diff
@@ -265,3 +278,43 @@ def __iter__(self) -> Iterator[T]:
     def raw_iterator(self) -> T:  # type: ignore[misc]
         for i in self.items:
             yield i
+
+class TravelingDataPipe:
```

Comment:
Could we rename it to something like `_DataPipeSerializationWrapper`? With `DataPipe` at the end of this class, we might give users a wrong hint. And, make it private (underscore) might be even better.

Reply:
I agree with the renaming, we might want to rename `TravelingIterDataPipe` and `TravelingMapDataPipe` too, especially if these names show up in the DataPipe graph and users want to know what they are